apps/fluent-tester/src/RNTester/TestComponents/Shimmer/ShimmerTest.tsx

```diff
@@ -0,0 +1,45 @@
+import * as React from 'react';
+import { Shimmer, Separator } from '@fluentui/react-native';
+import { Stack } from '@fluentui-react-native/stack';
+import { stackStyle } from '../Common/styles';
+import { SHIMMER_TESTPAGE } from './consts';
+import { Text, View, processColor } from 'react-native';
+import { commonTestStyles as commonStyles } from '../Common/styles';
+
+export const ShimmerTest: React.FunctionComponent<{}> = () => {
+  return (
+    <View>
+      <Text style={commonStyles.section} testID={SHIMMER_TESTPAGE}>
+        Shimmer Test Page
+      </Text>
+      <Separator />
+      <Stack style={stackStyle}>
+        <Shimmer
+          style={{ flex: 1, alignItems: 'center' }}
+          appearance={{
+            tintColor: processColor('#FF0000'),
```

Comment:
what determine this color?

Reply:
Random color used to test that the tintColor prop did indeed set. I suppose it would be better if this came from some theme somewhere instead